docs/recipes/modernjs.mdx

```diff
@@ -0,0 +1,105 @@
+import { ChromeExtension } from '../../components/extension';
+import { Image } from '../../components/ui/image';
+import { SourceCode } from '@theme';
+
+# ModernJS (Canary)
+
+<SourceCode href="https://github.com/ZephyrCloudIO/zephyr-examples/tree/main/examples/modernjs-app" />
+
+In this guide, we'll walk through creating a Modern.js application and configuring it with Zephyr's Canary release. 
+
+:::danger
+*Please note that this plugin is still in testing stage and Module Federation features are currently under development and will be available in future releases.*
+:::
+
+:::info Prerequisites
+
+- We expect you to have finished our [Get Started](/general/get-started) guide.
+- Have our [browser extension](https://chromewebstore.google.com/detail/zephyr-mission-control/liflhldchhinbaeplljlplhnbkdidedn) installed in your browser.
+- A registered account on [Zephyr Cloud](https://app.zephyr-cloud.io).
+- npm >=10
```

Comment:
yarn also works I'm assuming, since we use Yarn

Reply:
Maybe we can even remove it whatsoever, it just comes bundled with node and we most likely support versions of the other bundlers without isssue.